### ISSUE TITLE
Namespace issue with same service name

### ISSUE THREAD

**Issue by aglio** [bojand/grpc-caller#26]
I have some `.proto` files in same directory. These files have difference package name and same service name.
When this `grpc-caller` calls with `includeDirs` of `loadOptions` that return error `12 UNIMPLEMENTED: unknown service foo.BarService`.

So, I read the source code of this `grpc-caller`.
I found the following problem.

https://github.com/bojand/grpc-caller/blob/master/lib/index.js#L54

This `loaded` variable include nested object with namespace.
However, `grpc-caller` call method can not set namespace that means `grpc-inspect` does not work as intended.
The `grpc-inspect` merged same name services in here.
https://github.com/bojand/grpc-inspect/blob/master/lib/inspect.js#L92

I added the namespace as below and it worked fine.
```javascript
    const descriptor = gi(loaded['foo'])
```

I think that should add `namespace` option into caller of `grpc-caller`.
What do you think? Or am I doing something wrong?

**Comment by bojand** [comment]
Hmm it appears this may be issue with [`@grpc/proto-loader`](https://github.com/grpc/grpc-node/tree/master/packages/grpc-protobufjs)? Example:

`./dup/one.proto`:
```
syntax = "proto3";

package one;

service Greeter {
  rpc SayHello (HelloRequest) returns (HelloReply) {}
}

message HelloRequest {
  string name = 1;
}

message HelloReply {
  string message = 1;
}
```
Then add `./dup/two.proto`, with same content as `one.proto` but with `package two;` for package.

Then the following code:

```js
const protoLoader = require('@grpc/proto-loader')
const grpcLibrary = require('grpc')
const protoFileName = path.resolve(__dirname, './dup/one.proto')
const protoDir = path.resolve(__dirname, './dup')

const options = {
  includeDirs: [protoDir]
}

const packageDefinition = protoLoader.loadSync(protoFileName, options)
console.dir(packageDefinition, { depth: 3, colors: true })
const packageObject = grpcLibrary.loadPackageDefinition(packageDefinition)
console.dir(packageObject, { depth: 3, colors: true })
```

Output:

```
{}
{}
```

if we comment out `includeDirs` option, the output is:

```
{ 'one.Greeter':
   { SayHello:
      { path: '/one.Greeter/SayHello',
        requestStream: false,
        responseStream: false,
        requestSerialize: [Function: serialize],
        requestDeserialize: [Function: deserialize],
        responseSerialize: [Function: serialize],
        responseDeserialize: [Function: deserialize],
        originalName: 'sayHello' } } }
{ one:
   { Greeter:
      { [Function: ServiceClient] super_: [Function: Client], service: { SayHello: [Object] } } } }
```

which is something we would expect.

**Comment by aglio** [comment]
Thank you for reply!
But I do not think it is a problem of `@grpc/proto-loader`.
In my case like below. For example,

`./dup/one/hello.proto`:
```
syntax = "proto3";

package one;

import "two/hello.proto";

service Greeter {
  rpc SayHello (HelloRequest) returns (HelloReply) {}
}

message HelloRequest {
  string name = 1;
}

message HelloReply {
  two.Greet greet = 1;
}
```

`./dup/two/hello.proto`:
```
syntax = "proto3";

package two;

service Greeter {
  rpc SayHello (HelloRequest) returns (HelloReply) {}
}

message Greet {
  string message = 1;
}

message HelloRequest {
  string name = 1;
}

message HelloReply {
  Greet greet = 1;
}
```
`./client.js`:
```
const protoLoader = require('@grpc/proto-loader')
const grpcLibrary = require('grpc')
const path = require('path')
const protoFilePath = 'one/hello.proto' // NOTE: relative path from root
const protoDir = path.resolve(__dirname, './dup')

const options = {
  includeDirs: [protoDir]
}

const packageDefinition = protoLoader.loadSync(protoFilePath, options)
console.dir(packageDefinition, { depth: 3, colors: true })
const packageObject = grpcLibrary.loadPackageDefinition(packageDefinition)
console.dir(packageObject['one'], { depth: 3, colors: true })
```

the output is:
```
{ 'one.Greeter':
   { SayHello:
      { path: '/one.Greeter/SayHello',
        requestStream: false,
        responseStream: false,
        requestSerialize: [Function: serialize],
        requestDeserialize: [Function: deserialize],
        responseSerialize: [Function: serialize],
        responseDeserialize: [Function: deserialize],
        originalName: 'sayHello' } },
  'two.Greeter':
   { SayHello:
      { path: '/two.Greeter/SayHello',
        requestStream: false,
        responseStream: false,
        requestSerialize: [Function: serialize],
        requestDeserialize: [Function: deserialize],
        responseSerialize: [Function: serialize],
        responseDeserialize: [Function: deserialize],
        originalName: 'sayHello' } } }

{ Greeter:
   { [Function: ServiceClient]
     super_: [Function: Client],
     service:
      { SayHello:
         { path: '/one.Greeter/SayHello',
           requestStream: false,
           responseStream: false,
           requestSerialize: [Function: serialize],
           requestDeserialize: [Function: deserialize],
           responseSerialize: [Function: serialize],
           responseDeserialize: [Function: deserialize],
           originalName: 'sayHello' } } } }
```

If the above mentioned, `grpc-caller` can not set `one` like `packageObject['one']`.
But, it works fine when I try use pure `@grpc/proto-loader` like above code.
Probably, the empty object that you shown has mistake. You should not set absolute path like `const protoFileName = path.resolve(__dirname, './dup/one.proto')` with `includeDirs`.
If you change this to relative path, you should know what I mean.
Thank you.

**Comment by bojand** [comment]
Hello, I think this should be fixed in [0.10.0](https://github.com/bojand/grpc-caller/releases/tag/v0.10.0). Feel free to reopen if still present.
